models/create_attribute.go

```diff
@@ -22,11 +22,10 @@ type CreateAttribute struct {
 	// enumeration
 	Enumeration CreateAttributeEnumeration `json:"enumeration"`
 
-	// Type of the attribute ( type 'id' only available if the category is 'transactional' attribute & type 'category' only available if the category is 'category' attribute )
-	// Required: true
-	Type *string `json:"type"`
+	// Type of the attribute. Use only if the attribute's category is normal, category or transactional ( type 'id' only available if the category is 'transactional' attribute & type 'category' only available if the category is 'category' attribute )
+	Type string `json:"type,omitempty"`
```

Comment:
@pierrre Since it was mandatory earlier but not now (after spec changes), we have to go with this change & i think not to worry as the API to create attribute got released yesterday only.